spec/cspec/c/gen-config-thy.py

```diff
@@ -208,14 +208,32 @@ def parse_gen_config(gen_config_file: str) -> Dict[str, str]:
     return config
 
 
+def parse_physBase(L4V_ARCH: str, devices_gen_h_file: str) -> str:
```

Comment:
if you want to follow python standards, `L4V_ARCH` as an argument name is a bit sus

Reply:
Happy to change. Any suggestions? Just lower case `l4v_arch`?

Reply:
(have done that now)